README.md

```diff
@@ -30,62 +30,79 @@ For the setup, it is assumed that there are multiple LND nodes running with conn
-    ...
-    "route_hints": [
-        {
-            "hop_hints": [
-                {
-                    "node_id": "020723c6f2203f1072336bd0e71bf4d11e367ab0b3010ce60c080abef0d4770db8",
-                    "chan_id": "12345",
-                    "fee_base_msat": 0,
-                    "fee_proportional_millionths": 0,
-                    "cltv_expiry_delta": 40
-                }
-            ]
-        },
-        {
-            "hop_hints": [
-                {
-                    "node_id": "026ff75cb2ff49b864833aa3c93970069070231a9ad64819252e190406dd0a6976",
-                    "chan_id": "12345",
-                    "fee_base_msat": 0,
-                    "fee_proportional_millionths": 0,
-                    "cltv_expiry_delta": 40
-                }
-            ]
-        }
-    ],
-    "payment_addr": "8f135e331fed1858e467f775fa644751a2b0da17ee0676ebb1bc8a994d667be6",
-    "num_msat": "6000",
-    "features": {
-        ...
-    }
-}
-```
+  ```json
+  {
+      "destination": "03422175ba6fed348de4f273cf81627c26d5ab2a78bfdc1a39e6f9c06354dd9371",
+      "payment_hash": "fd3b0f9a077006697ba8f82cc5673d1511cb11d1d1012662bf0c0b3c93f4245e",
+      ...
+      "route_hints": [
+          {
+              "hop_hints": [
+                  {
+                      "node_id": "020723c6f2203f1072336bd0e71bf4d11e367ab0b3010ce60c080abef0d4770db8",
+                      "chan_id": "12345",
+                      "fee_base_msat": 0,
+                      "fee_proportional_millionths": 0,
+                      "cltv_expiry_delta": 40
+                  }
+              ]
+          },
+          {
+              "hop_hints": [
+                  {
+                      "node_id": "026ff75cb2ff49b864833aa3c93970069070231a9ad64819252e190406dd0a6976",
+                      "chan_id": "12345",
+                      "fee_base_msat": 0,
+                      "fee_proportional_millionths": 0,
+                      "cltv_expiry_delta": 40
+                  }
+              ]
+          }
+      ],
+      "payment_addr": "8f135e331fed1858e467f775fa644751a2b0da17ee0676ebb1bc8a994d667be6",
+      "num_msat": "6000",
+      "features": {
+          ...
+      }
+  }
+  ```
 
 * It should be possible to pay this invoice as long as at least one node in the cluster is available.
 
 * When the payment comes in, an event is generated. Events can be listened for via:
 
-  `grpcurl -plaintext -v -d '{"hash":"EgR0u/TAjFXNGId9PEHVnryFl1sy/UwB2uN+YxC8S80="}' localhost:19090 lnmux.Service.SubscribeSingleInvoice`
+  ```bash
+  grpcurl -plaintext -v localhost:9190 lnmux.Service.SubscribeInvoiceAccepted
+  ````
 
-  (replace base64 hash)
+  Each time an invoice is accepted, you will receive a pair (hash , setID):
+  
+  ```json
+  Response contents:
+  {
+    "hash": "rCdcdDvpo7oqkO1Gkh1hikofseQpUZhgAIaw6gYXz7M=",
+    "setId": "J1q7YDvixChKrJjjP/yvv16KiQdpHqwW7gy8jqxDAvI="
+  }
+  ```  
```

Comment:
NIce updates here. Would do them in a separate commit, independent of the docker additions.